packages/web-ui-registration/src/RegisterFormDisabled.tsx

```diff
@@ -0,0 +1,37 @@
+import { ButtonGroup, Callout } from '@rocket.chat/fuselage';
+import { Form, ActionLink } from '@rocket.chat/layout';
+import { useSetting, useTranslation } from '@rocket.chat/ui-contexts';
+import type { ReactElement } from 'react';
+import { Trans } from 'react-i18next';
+
+import type { DispatchLoginRouter } from './hooks/useLoginRouter';
+
+export const RegisterFormDisabled = ({ setLoginRoute }: { setLoginRoute: DispatchLoginRouter }): ReactElement => {
+	const linkReplacementText = String(useSetting('Accounts_RegistrationForm_LinkReplacementText'));
```

Comment:
we can deprecate this setting, completely unnecessary and untranslatable: #25340

Reply:
we should but not sure if its recommendable right now